extensions/api/data-management/contractnegotiation/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/contractnegotiation/model/NegotiationInitiateRequestDto.java

```diff
@@ -14,7 +14,10 @@
 
 package org.eclipse.dataspaceconnector.api.datamanagement.contractnegotiation.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.validation.constraints.AssertTrue;
 import jakarta.validation.constraints.NotNull;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Where does this come from? We should be extremely careful about pulling in apache commons, and I honestly think its not necessary to pull it in, just to check a few strings for nullity.

Reply:
[update]: Apache Commons Lang gets pulled in by the Swagger Jakarta integration and Rest Assured. However, I still think we should severely restrict the use of it in our code base.

Reply:
I moved the `isValid` method from here:
https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/501883cc32f7d390b1c3bbad562c551210dbfa88/extensions/api/data-management/contractnegotiation/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/contractnegotiation/ContractNegotiationApiController.java#L175

Reply:
ok, then we should go through the code and cut out all those Apache Commons occurrences. can be done later though.